### DISCUSSION TITLE
feedback(list-item): disable clicks on internal elements

### DISCUSSION THREAD

**Discussion by VandeurenGlenn** [material-components/material-web#4816] [Ideas]
When getting the clicked target using composedPath returns internal elements.
Not really a big deal but think it would be more dev friendly to return the item itself.

<img width="554" alt="image" src="https://github.com/material-components/material-web/assets/8685280/941732f2-3f6b-407e-a9af-20e87e4b1e33">

**Comment by asyncliz** [comment]
I believe this is WAI for `event.composedPath()`, which returns the [internal shadow root nodes with `mode: 'open'` shadow roots](https://developer.mozilla.org/en-US/docs/Web/API/Event/composedPath).

Use [`event.target`](https://developer.mozilla.org/en-US/docs/Web/API/Event/target) to get a reference to the host element itself.

**Reply by VandeurenGlenn** [reply]
'event.target' won't return the clicked list-item because the click listener is the host element that gets the event vs setting a listener for each list-item

Disabling the internal clicks would also help with #4858
